server/api/scraper.py

```diff
@@ -0,0 +1,49 @@
+import re
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from webdriver_manager.chrome import ChromeDriverManager
+from webdriver_manager.utils import ChromeType
+
+class ScrapeAmazon:   
+    def __init__(self, URL):
+        chrome_options = Options()
+        chrome_options.add_argument('--no-sandbox')
+        chrome_options.add_argument('--ignore-certificate-errors')
+        chrome_options.add_argument('--headless')
+        chrome_options.add_argument('--disable-dev-shm-usage')
+        chrome_options.add_argument('--remote-debugging-port=9222')
+        executable_path=ChromeDriverManager(chrome_type=ChromeType.CHROMIUM).install()
+        
+        userAgent = 'Mozilla/5.0 (X11; Linux x86_64)' +  'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.39 Safari/537.36'
+        chrome_options.add_argument("user-agent={}".format(userAgent))   
+
+        driver = webdriver.Chrome(executable_path=executable_path, options=chrome_options)    
+        driver.get(URL)
+        self.shopURL = URL
+        self.title = self.getTitle(driver) 
+        self.oldPrice = self.getOldPrice(driver)  
+        self.price = self.getPrice(driver)  
+        self.imgURL = self.getImgURL(driver)  
+        self.availability = self.getAvailability(driver)  
+        driver.quit()
+    def getTitle(self, driver):
```

Comment:
Python style best practice is snake_case for function names, so get_title, get_old_price, etc.